source/loader/windows/platform_discovery_win.cpp

```diff
@@ -21,22 +21,14 @@
 
 namespace loader {
 
-static const char *knownAdaptersNames[] = {
-    MAKE_LIBRARY_NAME("ur_null", UR_VERSION),
-};
-
 std::vector<PlatformLibraryPath> discoverEnabledPlatforms() {
     std::vector<PlatformLibraryPath> enabledPlatforms;
 
     // UR_ADAPTERS_FORCE_LOAD  is for development/debug only
     char *altPlatforms = nullptr;
     _dupenv_s(&altPlatforms, NULL, "UR_ADAPTERS_FORCE_LOAD");
 
-    if (altPlatforms == nullptr) {
-        for (auto libName : knownAdaptersNames) {
-            enabledPlatforms.emplace_back(libName);
-        }
-    } else {
```

Comment:
Unfortunately I had to remove `knownAdaptersNames` from the windows build as the MSVC compiler is quite picky about zero-length arrays. Perhaps I should remove from linux equivalent too?

Reply:
You could use std::array. zero-length std:array is well defined so Windows should not complain. But if we don't have any adapters yet removing it will work as well I guess.

Reply:
Yea, I wouldn't worry about removing this - this code needs to be refactored anyway.